src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/health/TypedHealthMonitor.scala

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.dsde.agora.server.dataaccess.health
+
+import akka.actor.typed._
+import akka.actor.typed.scaladsl._
+import akka.actor.typed.scaladsl.adapter._
+import org.broadinstitute.dsde.workbench.util.health.{HealthMonitor, StatusCheckResponse}
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
+
+/**
+ * Akka Typed wrapper around the Workbench Health Monitor.
+ * Runs its own schedule sending messages to itself, instead of requiring others to send in messages.
+ */
+object TypedHealthMonitor {
+
+  //@formatter:off
+  sealed trait Command
+  private case object Check extends Command
+  case class GetCurrentStatus(replyTo: ActorRef[StatusCheckResponse]) extends Command
+
+  private case object CheckSingleTimer
+  private case object CheckRateTimer
+  //@formatter:on
+
+  def apply(healthMonitorSubsystems: HealthMonitorSubsystems,
+            initialDelay: FiniteDuration,
+            fixedRate: FiniteDuration,
+            dispatcherName: String): Behavior[Command] = {
+    Behaviors.setup { context =>
+      Behaviors.withTimers { timers =>
+        implicit val executionContext: ExecutionContext = context.executionContext
+        val healthMonitor = context.actorOf(
+          HealthMonitor
+            .props(healthMonitorSubsystems.subsystems)(healthMonitorSubsystems.checkHealth)
+            .withDispatcher(dispatcherName)
+        )
+
+        timers.startSingleTimer(CheckSingleTimer, Check, initialDelay)
+        timers.startTimerAtFixedRate(CheckRateTimer, Check, fixedRate)
```

Comment:
why do we need both `startSingleTimer` and `startTimerAtFixedRate`? If we're trying to add a pause prior to the  first `Check`, then maybe the `startSingleTimer` sends a different message e.g. `ChecksAreAGo`, and the receive for the `ChecksAreAGo` contains the call to  `startTimerAtFixedRate`?

Also, from the javadoc:
```
Warning: startTimerAtFixedRate can result in bursts of scheduled messages after long garbage collection pauses, which may in worst case cause undesired load on the system. Therefore startTimerWithFixedDelay is often preferred.
```

Reply:
This was a rough attempt to port the `classic` actor "behavior" as close as possible to the `typed` actors.

Both the [`classic`](https://doc.akka.io/api/akka/2.6/akka/actor/Scheduler.html) and [`typed`](https://doc.akka.io/api/akka/2.6/akka/actor/typed/Scheduler.html) scheduler-api-for-generic-`Runnable`s both have an initial delay option. However, [`withTimers(TimerScheduler)`](https://doc.akka.io/api/akka/2.6/akka/actor/typed/scaladsl/TimerScheduler.html) is the new recommended way to send messages-to-self... But `withTimers` doesn't contain an initial delay option, [seemingly out of design not omission](https://discuss.lightbend.com/t/akka-typed-schedulers-does-not-have-initial-timeout/1717). 🤷‍♀

In a PR update, I no longer delay the initial message. I also added supervision to restart both Actors, [as `typed` actors do not automatically restart like `classic` actors do](https://doc.akka.io/docs/akka/current/typed/from-classic.html#supervision). I'm assuming these actors three/five second delay was to avoid rapid-messages-during-crashing-restarts under Akka classic's supervision. My PR update with supervision should also operate with a delay. 🤞